client/state/plugins/wporg/test/actions.js

```diff
@@ -73,7 +73,7 @@ describe( 'WPorg Data Actions', () => {
 
 	test( 'FetchPluginList action should dispatch two actions and make a request', async () => {
 		await store.dispatch( fetchPluginsList( 'new', 1, undefined ) );
-		expect( store.dispatch ).toHaveBeenCalledTimes( 2 );
+		expect( store.dispatch ).toHaveBeenCalledTimes( 3 );
```

Comment:
Why does it need to call 3 actions now?  
If this is the right action the test description also needs to be updated.

Reply:
Yes it needs 3 times now. I have updated the test description.